COPYRIGHT

```diff
@@ -1 +1,5 @@
-Copyright (C) 2015 The Nightscout Foundation, http://www.nightscoutfoundation.org
```

Comment:
Oh!   Sorry about that!  I was really hoping you weren't going to ask me to fix the pull request.  Lol. Clearly, I have no idea what I am doing.   

Patricia Willis
www.plwillis.com


> On Jan 20, 2015, at 1:15 PM, Sulka Haro <notifications@github.com> wrote:
> 
> In COPYRIGHT:
> 
> > @@ -1 +1,5 @@
> > -Copyright (C) 2015 The Nightscout Foundation, http://www.nightscoutfoundation.org
> LOL right - you've accidentally created a pull request for the developers. Please disregard the rest of the comments! Apologies for comment spam. :D
> 
> —
> Reply to this email directly or view it on GitHub.
>